fontc/src/lib.rs

```diff
@@ -1018,11 +1021,12 @@ mod tests {
                 .collect();
             assert_eq!(
                 vec![
-                    (0x002C, 1),
-                    (0x002E, 0),
-                    (0x0030, 2),
-                    (0x031, 3),
-                    (0x032, 4)
+                    (0x0000, 0),
```

Comment:
hm .notdef should not be mapped to anything, 0x0000 is the NULL character which is something else, a control character that is no longer required to be defined (there used to be a recommendation in the TrueType spec about the first 4 glyphs which was removed later on, so nowadays a font no longer needs to have a glyph for NULL 0x0000 character).